src/components/Book.js

```diff
@@ -1,25 +1,27 @@
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/bookSlice';
-// import "../styles/Book.css";
+import '../styles/Book.css';
```

Comment:
- [ ] Good job so far. Your design needs a little touch to make it perfect.

According to the previous reviewer, you provided a very thick border on the book card which is not in the design. Kindly modify the styles so that it matches the provided design 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/224518033-6bcb00ec-0dee-48b5-9080-a9a7847cd4f2.png)

Zeplin design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/224518063-24863fba-8c0f-4ba6-96c2-c442f104ca49.png)